.idea/PiggyCustomEnchants.iml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why delete the IDE files ;-;

Reply:
problem with the gitignore?

Reply:
I didn't even touch the .idea folder, it just vanished!